core/src/impl/Kokkos_Profiling_Interface.hpp

```diff
@@ -157,6 +159,9 @@ void beginDeepCopy(const SpaceHandle dst_space, const std::string dst_label,
                    const std::string src_label, const void* src_ptr,
                    const uint64_t size);
 void endDeepCopy();
+void beginSync(const uint32_t deviceID, const std::string name,
```

Comment:
I think this should be the same order of arguments as parallel for? I mean we may want to at least move towards a more coherent interface (even if the current one isn't completely).

Reply:
Done